packages/mgt-components/README.md

```diff
@@ -55,6 +55,158 @@ The components can be used on their own, but they are at their best when they ar
     <mgt-agenda group-by-day></mgt-agenda>
     ```
 
+
+## <a id="disambiguation">Disambiguation</a>
+
+MGT is built using [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components). Web components use their tag name as a unique key when registering within a browser. Any attempt to register a component using a previously registered tag name results in an error being thrown when calling `CustomElementRegistry.define()`. In scenarios where multiple custom applications can be loaded into a single page this created issues for MGT, most notably in developing solutions using SharePoint Framework.
+
+To mitigate this challenge we built the [`mgt-spfx`](https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/main/packages/mgt-spfx) package. Using `mgt-spfx` developers could centralize the registration of MGT web components across all SPFx solutions deployed on the tenant. By reusing MGT components from a central location web parts from different solutions could be loaded into a single page without throwing errors. When using `mgt-spfx` all MGT based web parts in a SharePoint tenant used the same version of MGT.
```

Comment:
If we "deprecate" the `mgt-spfx` package, will it stay to that location?

Reply:
Good question. I've been thinking that we might not need to deprecate `mgt-spfx` as it still provides value even with disambiguation. And hey one day we might be able to deliver mgt-spfx@4 with a disambiguation baked in to run alongside v2 or v3

Reply:
@sebastienlevert I changed the tense used here to indicate that `mgt-spfx` is still an option, marking as resolved